inferentia/qa/setup_test_enviroment_and_test.sh

```diff
@@ -32,9 +32,9 @@ export TRITON_CORE_REPO_TAG=${DEFAULT_REPO_TAG}
 export TRITON_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
 export TRITON_THIRD_PARTY_REPO_TAG=${DEFAULT_REPO_TAG}
 export IDENTITY_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
-export PYTHON_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
+export PYTHON_BACKEND_REPO_TAG=jacky-inferentia-neuron
 export CHECKSUM_REPOAGENT_REPO_TAG=${DEFAULT_REPO_TAG}
-export TRITON_SERVER_REPO_TAG=${TRITON_SERVER_REPO_TAG:=${DEFAULT_REPO_TAG}}
+export TRITON_SERVER_REPO_TAG=jacky-inferentia-neuron
```

Comment:
same here

Reply:
Yes. This is just to ensure the test run links everything correctly. It is removed.